victims/data.py

```diff
@@ -24,6 +25,10 @@ def round_robin(*iterables):
 
 class Data:
     async def fetch(self, session, url):
+        if DEBUG:
+            session = aiohttp.ClientSession(
+                connector=aiohttp.TCPConnector(verify_ssl=False)
+            )
```

Comment:
I think this is not the proper place to handle this kwarg. The session is defined a few lines above and you're overwriting it. Beyond that now there's two different places defining sessions for the same request, what is a bit confusing. I'm gonna suggest something different around line 40;

```python
     async def fetch_spreadsheets(self):
        urls = (…)
        kwargs = {'loop': asyncio.get_event_loop()}
        if DEBUG:
            kwargs = {'connector': aiohttp.TCPConnector(verify_ssl=False)}
        
        async with aiohttp.ClientSession(**kwargs) as session:
            …
````